spec/spec.md

```diff
@@ -28,24 +28,27 @@ Participate:
 ------------------------------------
 
 ## Abstract
+  
+The JWT VC Presentation Profile defines a standardized way to interoperably exchange [[ref:Verifiable Credentials]] (VCs) between digital [[ref: Wallets]] and credential [[ref: Verifiers]].
 
-The JWT VC Presentation Profile defines a set of specifications to enable the interoperable presentation of Verifiable Credentials between Wallets and Verifiers/RPs.
+This document is **specification profile** not a specification. It is a collection of pre-existing specficiations with all but required optionality stripped from them with the purpose of ensuring interoperability.
```

Comment:
```suggestion
This document is not a specification, but a **profile**. It outlines which of the existing specifications are required to be used by implementations to interoperate among each other. It also clarifies mandatory to implement features for the optionalities allowed in the referenced specifications.
```

Reply:
i like the update but will reword a tiny bit